CRM/Core/Payment/PayJunction.php

```diff
@@ -57,6 +57,7 @@ public function __construct($mode, &$paymentProcessor) {
    *   the result in an nice formatted array (or an error object)
    */
   public function doDirectPayment(&$params) {
+    require_once 'PayJunction/pjClasses.php'; // necessary for event registrations 
```

Comment:
I have seen that there are lot of other payment classes like firstData, eWay etc. which has require_once inside doDirectPayment() . All though putting require_once at the top works for me